main.cpp

```diff
@@ -188,23 +222,35 @@ void AquaPark::relaxMuscles(int swimmingIntensity)
         goToSauna = true;
         std::cout << "You should use one of the sauna's to relax your muscles!" << std::endl;
     }
+    else
+        std::cout << "Don't need to use a sauna." << std::endl;
```

Comment:
If you use curly braces in the `if` part of an `if/else`:
    use them in the `else` part too.
- [ ] revise